hapi-fees/src/main/java/com/hedera/services/usage/token/TokenBurnUsage.java

```diff
@@ -31,8 +31,9 @@
 
 public class TokenBurnUsage extends TokenTxnUsage<TokenBurnUsage> {
 	private SubType currentSubType;
+	private int serialNumsCount = 0;
 
-	private TokenBurnUsage(TransactionBody tokenBurnOp, TxnUsageEstimator usageEstimator) {
+	public TokenBurnUsage(TransactionBody tokenBurnOp, TxnUsageEstimator usageEstimator) {
```

Comment:
Any reason change this to public instead of using the factory method?

Reply:
@ljianghedera, `BaseOperationUsage` needs to construct a new instance of `TokenBurnUsage`